cmd/utils/cmd.go

```diff
@@ -42,6 +43,26 @@ const (
 
 var logger = log.NewModuleLogger(log.CMDUtils)
 
+// Fatalf formats a message to standard error and exits the program.
+// The message is also printed to standard output if standard error
+// is redirected to a different file.
+func Fatalf(format string, args ...interface{}) {
+	w := io.MultiWriter(os.Stdout, os.Stderr)
+	if runtime.GOOS == "windows" {
```

Comment:
I have a question.
Is klaytn going to support Windows?
If not, I'll remove this code.

Reply:
I guess we do not need to consider this anymore